src/core/mod.rs

```diff
@@ -63,29 +63,29 @@ impl Deref for KyberSecretKey {
     }
 }
 
-type Subkey = (Option<KyberSecretKey>, R25519PrivateKey);
-
+pub(super) type PublicSubkey = (Option<KyberPublicKey>, R25519PublicKey);
```

Comment:
Let your code breath ^^

```suggestion
pub(super) type PublicSubkey = (Option<KyberPublicKey>, R25519PublicKey);

```